bentoml/serve.py

```diff
@@ -262,6 +262,8 @@ def serve_production(
                         "$(CIRCUS.WID)",
                         "--worker-env-map",
                         json.dumps(runner.scheduled_worker_env_map),
+                        "--prometheus-dir",
+                        prometheus_dir,
```

Comment:
Is this needed to make runner metrics work?

Reply:
yes. Runners might be multi-processing, too